celery_with_django/run_celery.bat

```diff
@@ -0,0 +1,13 @@
+@echo off
+
+rem Activate virtual environment
+call D:\ReverseBits\Projects\django\celery_with_django\env\Scripts\activate.bat
```

Comment:
this should be picked up from Env as this same folder structure won't be available to others, or it should be a relative path for more dynamic calling

Reply:
I tried this but doesn't work.

Reply:
Open a github issue regarding this, I am resolving the conversation